AccessCheckoutSDK/AccessCheckoutSDK/client/AccessCheckoutClient.swift

```diff
@@ -64,4 +64,26 @@ public struct AccessCheckoutClient {
             }
         }
     }
+
+    /**
+     Initialises the validation for card details using the provided validation configuration.
+    
+     This method sets up the validation logic for the UI fields specified in the configuration.
+     It passes accessCheckoutClient as an argument to validationInitialiser to prevent memory leaks, rather than storing it as a property.
+    
+     - Parameter validationConfiguration: The `ValidationConfig` that specifies which fields to validate,
+                                         the validation rules to apply, and the delegate to notify of validation events
+     */
+    public func initialiseValidation(_ validationConfiguration: ValidationConfig) {
+        let validationInitialiser = AccessCheckoutValidationInitialiser()
```

Comment:
Can we move the `validationInitialiser` to be stored at class level rather than initiated in this method?

The reason is that the card brands/validation config is fetched when the `AccessCheckoutValidationInitialiser` is instantiated. If we use different instances then we will unnecessarily fetch the card card brands/validation config for each instance.